client/homebrew/editor/snippetbar/snippetbar.jsx

```diff
@@ -70,7 +70,9 @@ const Snippetbar = createClass({
 	mergeCustomizer : function(oldValue, newValue, key) {
 		if(key == 'snippets') {
 			const result = _.reverse(_.unionBy(_.reverse(newValue), _.reverse(oldValue), 'name')); // Join snippets together, with preference for the child theme over the parent theme
-			return _.filter(result, 'gen'); //Only keep snippets with a 'gen' property.
+			return _.filter(result, function(snip) {
+				return(snip.hasOwnProperty('gen') || snip.hasOwnProperty('subsnippets'));
+			});
```

Comment:
Any downside if we just simplify this to:

```suggestion
			return result.filter(snip.gen || snip.subsnippets);
```
?

.gen and .subsnippets have to exist *and* be truthy (no empty strings for .gen, etc.)